cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json

```diff
@@ -542,6 +542,16 @@
 						}
 					]
 				},
+				{
+					"id": "excludeInterior",
```

Comment:
This isn't a scope type; it's a modifier. Not sure if we have a section for that?